package.json

```diff
@@ -15,8 +15,8 @@
     },
     "license": "SEE LICENSE IN LICENSE",
     "scripts": {
-        "dev": "http-server ./ -p 7000 -c-1 --cors",
-        "server": "http-server ./ -p 7000 --cors",
+        "dev": "http-server ./ -c-1 --cors",
+        "server": "http-server ./ --cors",
```

Comment:
removing the dedicated port, will always choose the next open port on your machine. 
The updates in the readme files mention `8900`, which might not always be the port chosen on a user's machine. The same is the case for the charts and editors that use vite. 

That said, the removal here is correct, just the readme files need to reflect that somehow.

Reply:
The README change "suggests" to force the port to 8900 via `PORT=8900 npm run ...`.
(`http-server` uses the env var if it's defined - it chooses the next open port only if `$PORT` is not defined).

Reply:
sweet - thanks!